run-tests.sh

```diff
@@ -11,6 +11,18 @@ export PATH=$PREFIX/bin:$PATH
 
 : ${ARCHS:=${TOOLCHAIN_ARCHS-i686 x86_64 armv7 aarch64}}
 
+if [ -z "$RUN_X86" ]; then
+    case $(uname) in
+    MINGW*)
+        RUN_X86=
```

Comment:
As the condition for running tests further down in the file is this
```
        if [ -n "$RUN" ]; then
            $RUN $file
```
it turns out that this change makes so that the tests aren't executed at all, if one tries to run them on actual windows. That's probably not intended?

Not sure of what would be a better way though, maybe a special value `RUN_X86=native`, and then extending the run code to this:

```
        if [ -n "$RUN" ]; then
            if [ "$RUN" = "native" ]; then
                ./$file
            else
                $RUN $file
            fi
        fi
```